apps/system/style/cards_view/cards_view.css

```diff
@@ -4,10 +4,19 @@
   top: 0;
   left: 0;
   width: 100%;
-  height: 100%;
+  height: calc(100% + 1.5rem);
```

Comment:
we need to add the 1.5rem to the rule for when the software home button is displayed too.
currently we can see the scrollbar with the SHB on.

We can also add a comment explaining why we're adding the 1.5rem :)